src/components/Claim/Generic/AlreadyClaimed.view.tsx

```
@@ -3,20 +3,81 @@
 import * as _consts from '../Claim.consts'
 import * as consts from '@/constants'
+import * as utils from '@/utils'
 import * as interfaces from '@/interfaces'
 import { useRouter } from 'next/navigation'
 import Link from 'next/link'
+import ConfirmDetails from '@/components/Global/ConfirmDetails/Index'
 
 export const AlreadyClaimedLinkView = ({ claimLinkData }: { claimLinkData: interfaces.ILinkDetails | undefined }) => {
     const router = useRouter()
+
+    const chainName = consts.supportedPeanutChains && consts.supportedPeanutChains.find((chain) => chain.chainId == claimLinkData?.chainId)?.name
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)